examples/mp-metrics-histogram/README_pt.adoc

```diff
@@ -0,0 +1,452 @@
+= Métricas com MicroProfile `@Histogram`
+:index-group: MicroProfile
+:jbake-type: page
+:jbake-status: published
+
+Este é um exemplo sobre como utilizar as métricas do MicroProfile no TomEE
+Este es un ejemplo sobre cómo utilizar las métricas de MicroProfile en TomEE.
+
+== Executando a aplicação:
+
+[source,bash]
+----
+$ mvn clean install tomee:run
+----
+
+Dentro da aplicação, há um endpoint que te vai da um histograma do clima das mais recentes temperaturas na cidade de Nova York.
+
+== Requisição:
+
+[source,bash]
+----
+$ curl -X GET http://localhost:8080/mp-metrics-histogram/weather/histogram
+----
+
+== Resposta:
+
+[source,javascript]
+----
+{
+    "count":15,
+    "max":55,
+    "mean":44.4,
+    "min":27,
+    "p50":45.0,
+    "p75":46.0,
+    "p95":54.0,
+    "p98":54.0,
+    "p99":54.0,
+    "p999":54.0,
+    "stddev":7.0710678118654755,
+    "unit":"degrees F"
+}
+----
+
+== Usando `@Histogram`
+
+As métricas do MicroProfile tem uma função que permite criar um histogram de dados.
+
+Para utilizar esta função, injetamos um objeto `MetricRegistry`, registrar o histogram e 
+Para utilizar esta función, injecta un objeto `MetricRegistry`, registra el Histograma, e adiciona dados ao histograma, como mostrado abaixo.
+
+[source,java]
+----
+@Inject
+private MetricRegistry registry;
+
+@Inject
+@Metric(name = "temperatures", description = "A histogram metrics example.",
+    displayName = "Histogram of Recent New York Temperatures")
+private Histogram histogram;
+
+@Path("/histogram")
+@GET
+@Produces(MediaType.APPLICATION_JSON)
+public Histogram getTemperatures() {
+    Metadata metadata = new Metadata("temperatures", MetricType.HISTOGRAM, "degrees F");
+    metadata.setDescription("A histogram of recent New York temperatures.");
+    final int[] RECENT_NEW_YORK_TEMPS = { 46, 45, 50, 46, 45, 27, 30, 48, 55, 54, 45, 41, 45, 43, 46 };
+    histogram = registry.histogram(metadata);
+    for(int temp : RECENT_NEW_YORK_TEMPS) {
+        histogram.update(temp);
+    }
+    return histogram;
+}
+----
+
+Existem algumas configurações definidas na anotação `@Histogram`:
+
+*String name* Opcional. Define o nome da métrica. Se não for fornecido
+explicitamente, o nome do objeto anotado é usado.
+
+*String displayName* Opcional. Um nome de exibição legível para metadados.
+
+*String description* Opcional. Uma descrição da métrica.
+
+*String[] tags* Opcional. Matriz de cadeia no formato = para fornecer
+etiquetas especiais para uma métrica.
+
+*boolean reusable* Indica se uma métrica com um determinado nome pode ser
+registre-se em mais de um local. Não se aplica ao `@Histogram`.
```

Comment:
The original text has: Does not apply to **gauges** or histograms. Here the citation to the gauges was omitted. The same problem is in Spanish text.

Change to "Não se aplica ao `@Histogram`" to "Não se aplica a gauges ou histogramas.". Need translate gauge to medidor? I dont know.
 
And in the original text not refer to object of type `@Histogram` but only the word histogram. The same problem is in Spanish text.

Reply:
done